pedestrian_publisher/launch/pedestrian_publisher_launch.py

```diff
@@ -0,0 +1,33 @@
+from launch import LaunchDescription
```

Comment:
actually this is not for this PR, but having here would change the call function in the carma-config (`/opt/carma/install instead of workspace/install`) . So let's create the PR for that.